README.md

```diff
@@ -31,15 +32,42 @@ internal class PlanetDbContext : DbContext
 }
 ```
 
-To get going with the DbContext:
+### Connection Options
 
+#### Option 1: Direct Database Connection
 ```csharp
 var mongoConnectionString = Environment.GetEnvironmentVariable("MONGODB_URI");
 var mongoClient = new MongoClient(mongoConnectionString);
 var db = PlanetDbContext.Create(mongoClient.GetDatabase("planets"));
 db.Database.EnsureCreated();
+var planet = db.Planets.FirstOrDefault(x => x.Name == "Earth");
 ```
 
+#### Option 2: Using Dependency Injection
+```csharp
+var nosqlConnectionString = builder.Configuration.GetConnectionString(ConnectionStringKey)!;
```

Comment:
Let's call this mongoConnectionString like the other example and maybe also not rely on ConnectionStringKey but put a name in there like MongoDbUri ?